aio/aio-proxy/aio_proxy/search/helpers/color.py

```diff
@@ -0,0 +1,31 @@
+import json
+import logging
+import os
+from urllib.request import urlopen
+
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
`is_env_dev` ?

Reply:
it's another var that I use 'color_url` from env variables, but I use the same file to load it.